worldwind/src/commonMain/kotlin/earth/worldwind/layer/RenderableLayer.kt

```diff
@@ -27,6 +32,8 @@ open class RenderableLayer @JvmOverloads constructor(displayName: String? = null
         require(index in renderables.indices) {
             logMessage(ERROR, "RenderableLayer", "setRenderable", "invalidIndex")
         }
+        val oldRenderable = renderables[index]
```

Comment:
could you replace this logic with:
return renderables.set(index, renderable).also { batchRenderers[it::class]?.removeRenderable(it) }

Tu avoid useless acces to the list.

Reply:
done###